products/BellHybrid/apps/application-bell-settings/models/DateTimeUnitsModel.cpp

```diff
@@ -12,37 +13,48 @@
 #include <service-time/Constants.hpp>
 #include <service-time/api/TimeSettingsApi.hpp>
 #include <service-time/service-time/TimeMessage.hpp>
+#include <widgets/DateSetSpinner.hpp>
 #include <widgets/TimeSetFmtSpinner.hpp>
 
 #include <ctime>
 
 namespace app::bell_settings
 {
-    TimeUnitsModel::TimeUnitsModel(app::ApplicationCommon *app) : application(app)
+    namespace
+    {
+        constexpr auto factoryResetDate   = date::year(2022) / date::month(8) / date::day(18);
```

Comment:
As @patrycja-paczkowska said, the default date should be 2022/01/01

Reply:
ok I did not remember the date I will change to 2022/1/1

Reply:
Fixed